app/src/main/java/com/android/voyageur/ui/trip/activities/ActivityItem.kt

```diff
@@ -90,21 +96,38 @@ fun ActivityItem(
                 }
 
                 Row(verticalAlignment = Alignment.CenterVertically) {
-                  // Delete icon, visible only when expanded
+                  // Edit and Delete icons, visible only when expanded
                   AnimatedVisibility(visible = isExpanded) {
-                    IconButton(
-                        onClick = { /*TODO: delete activity*/
-                          Toast.makeText(
-                                  context,
-                                  "Delete activity not implemented yet",
-                                  Toast.LENGTH_SHORT)
-                              .show()
-                        }) {
-                          Icon(
-                              imageVector = Icons.TwoTone.Delete,
-                              contentDescription = "Delete Activity",
-                              tint = Color.Red)
-                        }
+                    Row {
+                      IconButton(
+                          onClick = {
+                            navigationActions.navigateTo(Screen.EDIT_ACTIVITY)
+                            tripsViewModel.selectActivity(activity)
+                            //                              Toast.makeText(
+                            //                                  context,
+                            //                                  "Edit activity not implemented yet",
+                            //                                  Toast.LENGTH_SHORT)
+                            //                                  .show()
+                          }) {
+                            Icon(
+                                imageVector = Icons.TwoTone.Edit,
+                                contentDescription = "Edit Activity",
+                                tint = Color.Green)
```

Comment:
I think this color is a bit too bright for the rest of the app, and green isn't really suggestive for the Edit action. Could you leave the default color instead? (the same color as the expand icon for example)

Reply:
Changed it.